app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
@moolitayer Does `check_policy_prevent_callback` work?

I think I'm misreading the code before your change:

```ruby
    cb = {
      :class_name  => self.class.to_s,
      :instance_id => id,
      :method_name => :check_policy_prevent_callback,
      :args        => [cb_method, event_target],
      :server_guid => MiqServer.my_guid
    }

def check_policy_prevent_callback(*action, _status, _message, result)
end
```

I don't see how this callback can produce an `action, status, message,` and `result`. I only see 2 arguments. None of which are `event_target`. I must be mistaking how callback works.

I expected something closer to `:args => [*cb_method, nil, nil, event_target]`

Then this PR would proceed by changing `event_target` into class/id...